docs/sql-ref-ansi-compliance.md

```diff
@@ -380,6 +380,7 @@ When ANSI mode is on, it throws exceptions for invalid operations. You can use t
   - `try_element_at`: identical to the function `element_at`, except that it returns `NULL` result instead of throwing an exception on array's index out of bound.
   - `try_to_timestamp`: identical to the function `to_timestamp`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
   - `try_parse_url`: identical to the function `parse_url`, except that it returns `NULL` result instead of throwing an exception on url parsing error.
+  - `try_make_timestamp`: identical to the function `make_timestamp`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
```

Comment:
How about others `try_make_timestamp_ltz` and `try_make_timestamp_ntz`. Should we add them here too?

Reply:
added them.